client/js/components/NavigationDrawer.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-navigation-drawer
+    :value="showNav"
+    class="primary"
+    dark
+    app
+    data-cy="nav-drawer"
+    :permanent="showNav"
+    :mini-variant="isCollapsed"
+  >
+    <!-- Page Links -->
+    <v-list>
+      <v-list-item
+        v-for="({ text, icon, page }, i) in pageLinks"
+        :key="i"
+        link
+        exact
+        :to="page"
+        :data-nav="text"
+      >
+        <v-icon class="mr-4"> mdi-{{ icon }} </v-icon>
+        {{ text }}
+      </v-list-item>
+    </v-list>
+    <!-- Expand/Collapse -->
+    <template #append>
+      <v-list v-if="!isSmallDevice">
+        <v-list-item :data-cy="collapseMenuAttribute" @click="userHasCollapsed = !userHasCollapsed">
+          <v-icon class="mr-2">{{ collapseMenuIcon }}</v-icon>
+          <template v-if="!isCollapsed">Collapse Menu</template>
+        </v-list-item>
+      </v-list>
+    </template>
+  </v-navigation-drawer>
+</template>
+
+<script>
+export default {
+  name: 'NavigationDrawer',
+  data() {
+    return {
+      userHasCollapsed: false,
+    };
+  },
+  computed: {
+    authenticated() {
+      return this.$store.state.auth.authenticated;
+    },
+    pageLinks() {
```

Comment:
Always shows

1. Login/Logout
2. Rules
Then if logged in, show

3. Play
4. Stats